src/bci_build/package/gcc.py

```diff
@@ -76,13 +77,12 @@ def _is_main_gcc(os_version: OsVersion, gcc_version: _GCC_VERSIONS) -> bool:
         ),
         replacements_via_service=[
             Replacement(
-                regex_in_build_description="%%gcc_version%%",
+                regex_in_build_description="%%gcc_minor_version%%",
                 package_name=gcc_pkg,
                 parse_version=ParseVersion.MINOR,
-            )
+            ),
         ],
-        env={"GCC_VERSION": "%%gcc_version%%"},
-        additional_versions=["%%gcc_version%%"],
```

Comment:
Can't we just resurrect this line as follows? That should solve the missing build tags
```Python
        additional_versions=["%%gcc_minor_version%%"],
```

Reply:
done.